pages/common/sg.md

```diff
@@ -0,0 +1,20 @@
+# ast-grep
+
+> Tool for code structural search, lint, and rewriting.
```

Comment:
The title of the page should be the command name.

The full name can then be put in the description:
https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#descriptions
```suggestion
# sg

> Ast-grep is a tool for code structural search, lint, and rewriting.
```